11/alpine/Dockerfile

```diff
@@ -1,6 +1,13 @@
 FROM node:11.10.0-alpine
 
-RUN apk add --no-cache \
-    bash \
-    ca-certificates \
-    curl
+RUN echo @edge http://nl.alpinelinux.org/alpine/edge/community >> /etc/apk/repositories && \
```

Comment:
possible to aslo update the readme file ? :)

Reply:
Ops, you are right sorry!